### PR TITLE
upgrade structlog for py3.8

### PR DESCRIPTION
**Pull request by hackaugusto**
The old version of structlog does not work with python3.8

### CODE REVIEW

File: requirements/requirements-ci.txt [hunk 1]
```diff
@@ -76,7 +76,6 @@ jinja2==2.10.1
 jsonschema==3.0.1
 lazy-object-proxy==1.4.1
 lru-dict==1.1.6
-macholib==1.14            # via pyinstaller
```
Comment (hackaugusto) [inline requirements/requirements-ci.txt]: @ulope do we need this? I remember we had one thing that was removed unintentionally and was required for macos

Comment (karlb) [reply requirements/requirements-ci.txt]: AFAIK we should leave this in for exactly that reason.

**Comment by karlb** [comment]
New escape sequences in the log output break some tests.

**Comment by hackaugusto** [comment]
> New escape sequences in the log output break some tests.

There seems to be a bit more of problems going on, the

**Pull request closed without merging**